src/analysis/retail/warlock/affliction/modules/spells/ShadowEmbrace.tsx

```diff
@@ -30,12 +28,14 @@ class ShadowEmbrace extends Analyzer {
   };
   protected enemies!: Enemies;
 
+  BONUS_PER_STACK_BASE = this.selectedCombatant.hasTalent(TALENTS.DRAIN_SOUL_TALENT) ? 0.04 : 0.02;
```

Comment:
Usually talent multipliers are declared in the `constants.ts` file in your specs folder, and then imported. Helps make it more maintainable.

eg. 
```
export const SOME_TALENT_MULTIPLIER = 0.04;
export const SOME_OTHER_TALENT_MULTIPLIER = 0.02;
```
```suggestion
  BONUS_PER_STACK_BASE = this.selectedCombatant.hasTalent(TALENTS.DRAIN_SOUL_TALENT) ? SOME_TALENT_MULTIPLIER : SOME_OTHER_TALENT_MULTIPLIER;
```

Reply:
Thanks - implemented